documentation/asciidoc/topics/quickstart/infinispan-values.yaml

```diff
@@ -0,0 +1,20 @@
+#Build configuration
+images:
+  server: {server_image}:{server_image_version}
+  initContainer: registry.access.redhat.com/ubi8-micro
+#Deployment configuration
+deploy:
+  replicas: 2
+  container:
+    extraJvmOpts: ""
+    storage:
+      size: 1Gi
+      storageClassName: ""
+      ephemeral: false
+    resources:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 500m
+        memory: 512Mi
```

Comment:
As all of the `container` values are the defaults, I would limit this example to just `deploy.replicas:2`.